docs/index.md

```diff
@@ -1,71 +1,126 @@
-Open Data Services Sphinx Base
-==============================
+Standardising Interest Declarations
```

Comment:
If we're calling the whole thing declared. , then maybe this should read declared. | Standardising Interest Declarations?

Reply:
Yes.